packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -44,7 +44,7 @@ export default decorate([
       region: undefined,
       allowUnauthorized: undefined,
       useVhdDirectory: undefined,
-      encryptionKey: undefined,
+      encryptionKey: '',
```

Comment:
How is it related? :thinking:

Reply:
maybe it should be another PR, but I couldn't save without an encryption key 
![image](https://user-images.githubusercontent.com/50174/193264648-5debb511-833f-44b3-a2e8-e4bf177a806d.png)

Reply:
Where does the error comes from?